message_definitions/v1.0/common.xml

```diff
@@ -3587,6 +3587,11 @@
       <field type="float" name="size_y">Size in radians of target along y-axis</field>
     </message>
     <!-- MESSAGE IDs 180 - 229: Space for custom messages in individual projectname_messages.xml files -->
+    <message id="227" name="AOA_SSA">
+      <description>Angle of Attack and Side Slip Angle</description>
+      <field name="AOA" type="float">Angle of Attack (degrees)</field>
```

Comment:
I think we should add a 32 bit time_boot_ms, to make it easier to graph
most data mavlink msgs have a timestamp